test/source/tests/tests/decrypt.ts

```diff
@@ -183,6 +183,14 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithNewBrowser:
       await pubFrame.close();
     }));
 
+    ava.default('[standalone] - wrong message - checksum throws error', testWithNewBrowser(async (t, browser) => {
```

Comment:
here you could test with shared browser, because the test is not changing any settings. It will be faster.